idaes/models_extra/column_models/tests/test_plate_heat_exchanger.py

```diff
@@ -163,7 +162,8 @@ def test_dof(self, phe):
     @pytest.mark.component
     def test_initialize(self, phe):
         initialization_tester(
-            phe, duty=(245000, pyunits.W), optarg={"bound_push": 1e-8, "mu_init": 1e-8}
+            phe,
+            duty=(245000, pyunits.W),  # , optarg={"bound_push": 1e-8, "mu_init": 1e-8}
```

Comment:
@lxhowl Your PR should not have affected these tests - we need to look into the reason why these tests were failing a bit more to see why they were failing before doing anything to these (which might be in a separate PR).